src/tribol/mesh/CouplingScheme.cpp

```diff
@@ -595,6 +600,13 @@ bool CouplingScheme::isValidMethod()
         return false;
       }
     }
+
+    if ( this->m_contactMethod == SINGLE_MORTAR && this->m_useEnzyme ) {
+      // this is only needed on the nonmortar mesh (def'd as mesh2 for Enzyme mortar method)
```

Comment:
I think this is fine to add the comment here about only needing mesh2 ref. coords.